plugins/services/src/js/reducers/serviceForm/Container.js

```diff
@@ -1,37 +1,191 @@
-import {SET} from '../../../../../../src/js/constants/TransactionTypes';
+import {SET, ADD_ITEM, REMOVE_ITEM} from '../../../../../../src/js/constants/TransactionTypes';
+import {combineReducers} from '../../../../../../src/js/utils/ReducerUtil';
+import {JSONReducer as volumes} from './Volumes';
 import VolumeConstants from '../../constants/VolumeConstants';
+import ValidatorUtil from '../../../../../../src/js/utils/ValidatorUtil';
 
-let {MESOS} = VolumeConstants.type;
+import docker from './Docker';
 
-function container(state = {}, {type, path, value}) {
-  let joinedPath = path && path.join('.');
-  let newState = Object.assign({}, state);
-  if (!newState.type) {
-    newState.type = MESOS;
+const {MESOS} = VolumeConstants.type;
```

Comment:
At some point we should move `MESOS` and `DOCKER` constants from `VolumeConstants` to `ContainerConstants` or something